examples/internal_wave.jl

```diff
@@ -140,31 +140,67 @@ run!(simulation)
 # To visualize the solution, we load a FieldTimeSeries of w and make contour
 # plots of vertical velocity.
 
-using Printf, Plots
+using GLMakie
 
-w_timeseries = FieldTimeSeries("internal_wave.jld2", "w")
-Nt = length(w_timeseries.times)
-x, y, z = nodes(w_timeseries)
+filename = "internal_wave"
 
-# and makes an animation with Plots.jl:
+fig = Figure(resolution = (800, 400))
 
-anim = @animate for (i, t) in enumerate(w_timeseries.times)
+ax = Axis(fig[1, 1];
+          xlabel = "x",
+          ylabel = "z",
+          limits = ((-π, π), (-π, π)),
+          aspect = AxisAspect(1))
 
-    @info "Drawing frame $i of $Nt..."
+nothing #hide
 
-    w = interior(w_timeseries[i], :, 1, :)
+# We use Makie's `Observable` to animate the data. To dive into how `Observable`s work we
+# refer to [Makie.jl's Documentation](https://makie.juliaplots.org/stable/documentation/nodes/index.html).
 
-    contourf(x, z, w', title = @sprintf("ωt = %.2f", ω * t),
-                      levels = range(-1e-8, stop=1e-8, length=10),
-                       clims = (-1e-8, 1e-8),
-                      xlabel = "x",
-                      ylabel = "z",
-                       xlims = (-π, π),
-                       ylims = (-π, π),
-                   linewidth = 0,
-                       color = :balance,
-                      legend = false,
-                 aspectratio = :equal)
+iter = Observable(0)
+
+# We open the saved output `.jld2` files and extract the vorticity and speed.
+
+using JLD2
+
+file = jldopen(filename * ".jld2")
+grid = file["serialized/grid"]
+
+title = @lift(string("ωt = ",
+              string(round(file["timeseries/t/" * string($iter)] * ω, digits=3))))
+
+w = @lift(Array(file["timeseries/w/" * string($iter)][:, 1, :]))
```

Comment:
I think we should use `FieldTimeSeries` here rather than manually constructing things with `JLD2`, etc.

Reply:
Done!